fog/sample-paykit/src/client.rs

```diff
@@ -534,13 +533,16 @@ impl Client {
         Ok(res.num_blocks + self.new_tx_block_attempts as u64)
     }
 
-    /// Retrieve the currently configured minimum fee from the consensus service
-    pub fn get_fee(&mut self) -> Result<u64> {
-        Ok(self
-            .consensus_service_conn
-            .fetch_block_info()?
-            .minimum_fee_or_none(&Mob::ID)
-            .unwrap_or(0))
+    /// Retrieve the current last block info structure from consensus service.
+    /// This includes fee data and last block index
+    pub fn get_last_block_info(&mut self) -> Result<BlockInfo> {
+        Ok(self.consensus_service_conn.fetch_block_info()?)
+    }
+
+    /// Retrieve the currently configured minimum fee for a token id from the
+    /// consensus service
+    pub fn get_minimum_fee(&mut self, token_id: TokenId) -> Result<Option<u64>> {
+        Ok(self.get_last_block_info()?.minimum_fee_or_none(&token_id))
```

Comment:
I'm guessing the minimum fee returned here for the Mob (?) token is the Mob::MINIMUM_FEE? I took a brief look at the BlockInfo#minimum_fee_or_none, and it looks like there's a map that contains the minimum fees per tokens, but couldn't find what the actual minimum fee is for the Mob (?) token.

Reply:
It gets configured in consensus using tokens toml, at least in master now